rafs/src/metadata/layout/v6.rs

```diff
@@ -353,10 +352,7 @@ impl RafsV6SuperBlockExt {
         }
 
         let chunk_size = u32::from_le(self.s_chunk_size) as u64;
-        if !chunk_size.is_power_of_two()
-            || chunk_size < EROFS_BLOCK_SIZE
-            || chunk_size > RAFS_MAX_CHUNK_SIZE
```

Comment:
It would be safer to introduce a real "RAFS_MAX_CHUNK_SIZE" to cover up to 16M?

Reply:
Fixed